src/pages/dfi/_components/ExchangeSection.tsx

```diff
@@ -22,6 +22,7 @@ import BitmartLogo from "../../../../public/assets/svg/exchanges/logo-bitmart.sv
 import GateioLogo from "../../../../public/assets/svg/exchanges/logo-gateio.svg";
 import LOCKLogo from "../../../../public/assets/svg/exchanges/logo-lock.svg";
 
+// TODO: Remove this V1 file and all svg assets
```

Comment:
we can remove the v1 files already, not reusing any v1 UI when we launch this refresh

Reply:
I removed this file, but realised the SVGs are still used somewhere (BuyDFI)